src/avfoundation.cs

```diff
@@ -12289,5 +12583,21 @@ interface AVCapturePhoto
 
 		[Wrap ("AVCaptureDeviceTypeExtensions.GetValue (WeakSourceDeviceType)")]
 		AVCaptureDeviceType SourceDeviceType { get; }
+
+		// From @interface AVCapturePhotoBracketedCapture (AVCapturePhoto)
+
+#if !MONOMAC
```

Comment:
Could we go `[NoMac]` here? Are `AVCaptureBracketedStillImageSettings`  and `AVCaptureLensStabilizationStatus` just not available?

`SequenceCount` could definitely avoid the `#if !MONOMAC`

Reply:
AVCaptureBracketedStillImageSettings and AVCaptureLensStabilizationStatus is not present in the mac AFAIK. Moving the other ones out.

Reply:
Comment is misleading `AVCaptureLensStabilizationStatus`  seems to be present since you did not put it in the `#if !MONOMAC` and it built just fine (:

@spouliot @mandel-macaque should we take this approach instead? https://github.com/xamarin/xamarin-macios/blob/master/src/scenekit.cs#L76-L97

It's also a comment for the future, I'd like to know the approach we prefer, moving forward.

Reply:
Sorry, I meant just one of them. tested the others :)